src/config.rs

```diff
@@ -97,7 +120,9 @@ fn env_overrides() -> config::Config {
 }
 
 pub fn read_config() -> Result<UserSettings, ConfigError> {
-    let mut config_builder = config::Config::builder().add_source(env_base());
+    let mut config_builder = config::Config::builder()
+        .add_source(env_base())
+        .add_source(default_mergetool_config());
```

Comment:
nit: these should probably be swapped so environment variables override the defaults (doesn't matter right now since there's no overlap)